moment-range/index.d.ts

```diff
@@ -14,11 +14,8 @@ declare namespace momentRange {
         start: moment.Moment;
         end: moment.Moment;
 
-        constructor(range: string);
-        constructor(range: Date[]);
-        constructor(range: moment.Moment[]);
-        constructor(start: Date, end: Date);
-        constructor(start: moment.Moment, end: moment.Moment);
+        constructor(range: string | Date[] | moment.Moment[]);
+        constructor(start: Date | moment.Moment, end: Date | moment.Moment);
```

Comment:
Well, you are right. The change is not the same, as now you can mix Date and Moment types in start and end parameter. But if you check the moment-range implementation, it also works that way, so the definition is better:
`constructor(start, end) {
    let s = start;
    let e = end;

    if (arguments.length === 1 || end === undefined) {
      if (typeof start === 'object' && start.length === 2) {
        [s, e] = start;
      }
      else if (typeof start === 'string') {
        [s, e] = start.split('/');
      }
    }

    this.start = (s === null) ? moment(-8640000000000000) : moment(s);
    this.end = (e === null) ? moment(8640000000000000) : moment(e);
  }`